lib/lotus/commands/new.rb

```diff
@@ -27,6 +27,21 @@ def initialize(app_name, environment, cli)
       def start
         @command.start
       end
+
+      private
+      def _get_real_app_name(app_name)
+        case app_name
+        when '.'
+          ::File.basename(Dir.getwd)
+        when '/'
```

Comment:
@mengqing This is confusing for me: what we're trying to address here? Does makes sense to have an usage like `lotus new /`?